test/webserver.erl

```diff
@@ -134,12 +134,28 @@ listen(Module) ->
 
 accept(ssl, ListenSocket) ->
     {ok, Socket} = ssl:transport_accept(ListenSocket, 10000),
-    ok = ssl:ssl_accept(Socket),
+    ok = ssl_accept(Socket),
```

Comment:
This will crash as ssl_accept/1 returns the Socket.

Reply:
good point, fixed. 
I don't think it's used though if that didn't fail as part of the CI 🙊